### PR TITLE
Fix kb image version in the install script

### PR DESCRIPTION
**Pull request by JimmyYang20**

### CODE REVIEW

**Comment by llhuii** [comment]
you have rename federated-learning-surface-defect-detection-aggregation.Dockerfile to `federated-learning-surface-defect-detection-aggregation-v2.Dockerfile`

But the new file name does not occur in https://github.com/kubeedge/sedna/blob/main/examples/build_image.sh

File: pkg/globalmanager/runtime/storage_initializer_injector.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 
 const (
 	downloadInitalizerContainerName = "storage-initializer"
-	downloadInitalizerImage         = "kubeedge/sedna-storage-initializer:v0.3.0"
```
[the remark this answers is not in the document]
Comment (JimmyYang20) [reply pkg/globalmanager/runtime/storage_initializer_injector.go]: ok

File: examples/build_image.sh [hunk 2]
```diff
@@ -26,6 +26,8 @@ federated-learning-mistnet-yolo-aggregator.Dockerfile
 federated-learning-mistnet-yolo-client.Dockerfile
 federated-learning-surface-defect-detection-aggregation.Dockerfile
```
Comment (llhuii) [inline examples/build_image.sh:27]: In this commit `federated-learning-surface-defect-detection-aggregation.Dockerfile`/`federated-learning-surface-defect-detection-train.Dockerfile` have been renamed, so we need to remove this line from `build_image.sh`

File: examples/build_image.sh [hunk 3]
```diff
@@ -24,8 +24,8 @@ EXAMPLE_REPO_PREFIX=${IMAGE_REPO}/sedna-example-
 dockerfiles=(
 federated-learning-mistnet-yolo-aggregator.Dockerfile
 federated-learning-mistnet-yolo-client.Dockerfile
-federated-learning-surface-defect-detection-aggregation.Dockerfile
-federated-learning-surface-defect-detection-train.Dockerfile
+federated-learning-surface-defect-detection-aggregation-v2.Dockerfile
```
[the remark this answers is not in the document]
Comment (JimmyYang20) [reply examples/build_image.sh:27]: The content of the current FL Dockerfile is for FL-v2, and the initial FL Dockerfile is still reserved.

**Pull request merged**